pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -155,8 +155,11 @@ def render(self, task) -> Text:
                 self._current_task_id = task.id
             if self._trainer.training and task.id != self._current_task_id:
                 return self._tasks[task.id]
-            _text = ""
 
+            if self._trainer.progress_bar_callback.main_progress_bar_id != task.id:
+                return Text("")
```

Comment:
Should the condition above be changed instead of adding this `if` here?

```diff
-            if self._trainer.state.fn != TrainerFn.FITTING or self._trainer.sanity_checking:
+            if not self._trainer.training:
```

Reply:
I think if running with val_check_interval, then once the first evaluation loop is done, it won't update up until the next training batch is processed.

Reply:
Kept the condition, but joined with the block above.

Reply:
cool!